src/datepicker/jalali/ngb-calendar-persian.ts

```diff
@@ -0,0 +1,263 @@
+/**
+ * Created by mehrabisajad on 2017/03/28.
+ */
+import {NgbDate} from '../ngb-date';
+import {NgbPeriod} from '../ngb-calendar';
+import {Injectable} from '@angular/core';
+import {NgbCalendarJalali} from './ngb-calendar-jalali';
+
+function mod(a: number, b: number): number {
```

Comment:
Not sure I like the function name `mod` for this - in mathematics, `mod` is typically used to refer to the modulus operation, for which the closest analog in JS is the `%` operator.